resources/resource_ids.spec

```diff
@@ -182,4 +182,9 @@
     "META": {"sizes": {"includes": [250]}},
     "includes": [58750]
   },
+  # This file is generated during the build.
+  "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-brave_private_new_tab/brave_private_new_tab.grd": {
+    "META": {"sizes": {"includes": [250]}},
```

Comment:
nit: I don't think this webui will be using more than 1 ID (no external images or dynamic JS modules), so reserving 250 seems extreme. Perhaps put this to 20 at most.

Reply:
makes sense